docs/install/install-knative-gcp.md

```diff
@@ -60,28 +64,29 @@
         In order to make controller compatible with Workload Identity, use
         [ko](http://github.com/google/ko) to apply
         [controller-gke](../../config/core/deployments/controller-gke.yaml)
-        first.
+        first. It is in `config/core/deployments`.
```

Comment:
nit: since you already have the full ko command below, do we really state it here again?